content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,105 @@
+| **Preferred Availability Zone(s)** | (_GCP only_) The [availability zone](https://cloud.google.com/compute/docs/regions-zones/#available) for your selected region.<br/><br/>If you choose *Select zone(s)*, you must choose at least one zone from **Zone Suffix**.  You can choose up to three preferred zones. |
+| **Persistent Storage Encryption** | (_AWS only_) Whether data is encrypted when stored to disk.  When selected, this mounts [encrypted EBS volumes](https://aws.amazon.com/ebs/faqs/#Encryption) to your instance. |
+
+After selecting your preferences, choose **Next** to determine your size requirements.
+
+## Size settings
+
+The Sizing tab determines your subscription cluster requirements, which are determined by databases in your subscription and their requirement.
+
+When the Sizing tab first opens, it lets you enter your database details.
+
+| Setting | Description |
+|:---------|:-----------|
+| _New database settings:_|
+| **Name** | A custom name for your database (_required_) |
+| **Protocol** | _Redis_ or _Memcached_ |
+| **Memory Limit (GB)** | The size limit for the database, Specify small sizes as decimals of 1.0&nbsp;GB; example: `0.1` GB (minimum).| 
+| **Throughput By** | Identifies how throughput is measured for the database, either operations per second (_Ops/sec_) or _Number of shards_. |
+| **Modules** | Identifies a module used by the database.  Choose from [RedisSearch&nbsp;2](#), [RedisGraph](#), [RedisBloom](#), or [RedisTimeSeries](#).<br/><br/>If you select RedisSearch 2, you also need to specify a value for **Number of Documents**.  This defines the maximum internal array size ([MAXDOCTABLESIZE](https://oss.redislabs.com/redisearch/Configuring/?_ga=2.155176508.524468484.1612194154-499260268.1607530891#maxdoctablesize)).|
+| **Replication** | Enables an in-memeory replica, which in turn enables real-time failover, but doubles memory consumption. |
+| **Data Persistence** | Defines the data persistence policy, if any. See [Database Persistence]({{< relref "/rs/concepts/data-access/persistence.md" >}}) |
+| **OSS Cluster API support** | Allows client to use the OSS Cluster API to access the database. |
+| **Ops/sec** | The operations required per second (when **Throughput By** is set to _Ops/sec_) |
+| **Number of Shards** | The number of shards set aside for the database (when **Throughput By** is set to _Number of Shards_) |
+
+When finished, click the Save button to add the database to your subscription.
+
+{{</* note */>}}Once a database has been saved to your subscription, you cannot change it here.  If you make a mistake, either:
+
+- Delete it from this list and create a new one
+
+- Edit the database after creating the subscription.{{</* /note */>}}
+
+After saving your database, you can:
+
+- Increase the Quantity of databases matching your selected settings (&lt;plus/minus buttons from quantity field).
+
+- Add another set of database choices (&lt;image of blue plus button>).
```

Comment:
Maybe just: "Add another database to your subscription"

Reply:
Reworked